AdyenActions/UI/View Controllers/Await View Controller/AwaitView.swift

```diff
@@ -13,7 +14,7 @@ internal final class AwaitView: UIView {
     internal lazy var icon: UIImageView = {
         var image = UIImage(named: viewModel.icon)
         if image == nil {
-            image = UIImage(named: viewModel.icon, in: Bundle.coreInternalResources, compatibleWith: nil)
```

Comment:
This is interesting one.. 
If we have  BLIK component in **Components** but `blik` icon in **Actions** ?

Reply:
Yeah, because the `blik` icon is used in the await action component when the payment method is `blik`, same for MBWay.